package.json

```diff
@@ -1,33 +1,45 @@
 {
-  "name": "@parity/dapp-addresses",
-  "description": "Parity default addressbook view",
-  "version": "1.99.99",
+  "name": "bond-dapp-addresses",
+  "version": "1.0.0",
+  "description": "address dapp using bonds",
   "main": "index.js",
-  "author": "Parity Team <admin@parity.io>",
-  "maintainers": [],
-  "contributors": [],
-  "license": "GPL-3.0",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/paritytech/dapp-addresses.git"
-  },
-  "keywords": [],
   "scripts": {
     "build": "NODE_ENV=production webpack --colors --progress",
     "watch": "webpack --colors --progress --watch"
   },
+  "author": "Parity Team <admin@parity.io>",
+  "license": "GPL-3.0",
+  "dependencies": {
+    "@parity/api": "^2",
+    "@parity/dapp-accounts": "file:../dapp-accounts",
+    "@parity/ui": "file:../js-ui",
+    "bignumber.js": "^4.0.4",
+    "oo7": "^0.5.6",
+    "oo7-parity": "file:../oo7-parity",
+    "oo7-react": "file:../oo7-react",
+    "parity-reactive-ui": "file:../parity-reactive-ui",
+    "prop-types": "^15.6.0",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "react-intl": "^2.4.0",
+    "semantic-ui-react": "^0.74.2"
+  },
   "devDependencies": {
-    "babel-cli": "^6.22.2",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
     "babel-eslint": "^7.2.3",
     "babel-loader": "^7.0.0",
     "babel-plugin-react-intl": "2.3.1",
-    "babel-plugin-transform-class-properties": "6.23.0",
-    "babel-plugin-transform-decorators-legacy": "1.3.4",
-    "babel-plugin-transform-es2015-modules-commonjs": "6.24.1",
-    "babel-plugin-transform-object-rest-spread": "6.23.0",
-    "babel-plugin-transform-react-remove-prop-types": "0.3.2",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.3.3",
     "babel-preset-env": "^1.6.0",
-    "babel-preset-react": "^6.22.0",
+    "babel-preset-es2015": "6.22.0",
```

Comment:
Remove 2015/2016, this is deprecated in Babel. Only keep env as the replacement.